File: 0.2 Coordinates.c
#include <stdio.h>
char *point_in_quadrant(double x, double y){
    char *zero_res = "One of the coordinates is equal to zero!";
    char *res_1 = "I";
    char *res_2 = "II";
    char *res_3 = "III";
    char *res_4 = "IV";
    char *res_origin = "It's the origin!";

    if (x > 0 && y > 0) return res_1;
    else if (x < 0 && y > 0) return res_2;
    else if (x < 0 && y < 0) return res_3;
    else if (x > 0 && y < 0) return res_4;
    else if (x == 0 && y == 0) return res_origin;
    else return zero_res;
}

int main() {
    double x,y;
    scanf(" %lf", &x);
    scanf(" %lf", &y);
    char *quadrant = point_in_quadrant(x,y);
    printf("%s\n", quadrant);
    return 0;
}

File: 0.2 Coordinates.py
# Найдите точку на декартовой координатной плоскости: к какому из четырех квадрантов принадлежит точка?
# Считайте из ввода два числа, не обязательно целых, первое число будет указывать положение по оси X, второе - по оси Y.
# Давайте продолжим называть квадранты римскими цифрами, как показано на рисунке.

# Если точка является началом координат (0, 0), выведите «It's the origin!».
# Если точка лежит на осях, либо x = 0, либо y = 0, выведите «One of the coordinates is equal to zero!».

# Sample Input 1:
# 3.987
# -10
# Sample Output 1:
# IV

def decart(x, y):
    if x > 0 < y: return 'I'
    elif x < 0 < y: return 'II'
    elif x < 0 > y: return 'III'
    elif x > 0 > y: return 'IV'
    else: return "It's the origin!"
    
print(decart(float(input()), float(input())))
